rts/lra/lra-proxy/api/src/main/java/io/narayana/lra/client/internal/proxy/ProxyService.java

```diff
@@ -224,6 +226,16 @@ public String joinLRA(LRAParticipant participant, LRAParticipantDeserializer des
         }
     }
 
+    @Override
+    public void registerDeserializer(LRAParticipantDeserializer deserializer) {
+        deserializers.add(deserializer);
+    }
+
+    @Override
+    public void unregisterDeserializer(LRAParticipantDeserializer deserializer) {
```

Comment:
Might be worth a comment to why override method as empty

Reply:
Good spot, I have implemented the body of the method.

Reply:
great